core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -548,7 +574,7 @@ private StorageDirView allocateSpace(long sessionId, AllocateOptions options) {
         }
         if (options.isEvictionAllowed()) {
           LOG.debug("Free space for block expansion: freeing {} bytes on {}. ",
-                  options.getSize(), options.getLocation());
+              options.getSize(), options.getLocation());
```

Comment:
Why can the original code pass the Coding-style check?

Reply:
I rolled back these code style formatting.